bio/gatk/applybqsrspark/environment.yaml

```
@@ -0,0 +1,8 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - gatk4 ==4.2.0.0
+  - openjdk =8
+  - snakemake-wrapper-utils ==0.1.3
```

As in my other review, I'd pin less stringently, here:
```suggestion
  - gatk4 =4.2
  - openjdk =8
  - snakemake-wrapper-utils =0.1.3
```